docs/source/aws.rst

```diff
@@ -0,0 +1,25 @@
+.. _aws:
+
+AWS Overview
+============
+
+.. figure:: ElasticBLASTonAWS-architecture.png
+   :alt: Overview of ElasticBLAST at AWS
+   :class: with-border
+
+
+AWS Configuration
+=================
+
+The minimum required configuration parameters for running ElasticBLAST in AWS include:
+
+* :ref:`region <elb_aws_region>`
+* :ref:`subnet <elb_aws_subnet>`
+* :ref:`security-group <elb_aws_security_group>`
+
```

Comment:
It looks like key-pair is also required for 0.0.16.

Reply:
Documentation has been updated: 3236c61